src/display/editor/editor.js

```diff
@@ -1142,11 +1148,90 @@ class AnnotationEditor {
 
     bindEvents(this, this.div, ["pointerdown"]);
 
+    if (this.isResizable && this._uiManager._supportsPinchToZoom) {
+      this.#touchManager ||= new TouchManager({
+        container: this.div,
+        isPinchingDisabled: () => false, // !this.isSelected,
```

Comment:
This looks a little strange, since the comment looks like a piece of code accidentally left commented out.
Is this actually intended, or should the callback function be updated to return the contents of the comment instead?

Finally, since these callback functions are *optional* why even provide them if they just return `false`?